schema/edition_set.js

```diff
@@ -1,7 +1,10 @@
-import { isEmpty } from "lodash"
+import { isEmpty, includes } from "lodash"
 import { IDFields } from "./object_identification"
 import Dimensions from "./dimensions"
 import { GraphQLString, GraphQLBoolean, GraphQLObjectType } from "graphql"
+import { capitalizeFirstCharacter } from "lib/helpers"
+
+const EditionSetAvailabilities = ["sold", "on hold", "on loan", "permanent collection"]
```

Comment:
Is the idea of not including For Sale and Not for Sale here is because they are not displayed as availability string in UI but rather as price and something else if not for sale?

Reply:
That's correct!

Reply:
May be a bit more descriptive to be a bit more specific here, something along `EditionSetDisplayableAvailabilities` to give a bit more context why it's not all possible availabilities. But that is super minor obviously...